### ISSUE TITLE
API behaviour when SIM is not present.

### ISSUE THREAD

**Issue by tauzen** [svic/SecureElement-v0.1#27]
```seManager.getSEReaders()``` is rejected with ```SEGenericError``` when the SIM card is not present in the SIM slot. According to Open Mobile API spec we should list all the readers. We should be able to tell if SIM card is present from ```SEReader.isPresent``` property.

**Comment by psiddh** [comment]
Yes Agree. In fact this will be fixed in the latest push / patches. However we also have another issue to tackle in the same space as per Open Mobile API spec. But more about that issue later!
